### PR TITLE
further increase test coverage

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: Symfony/Sniffs/NamingConventions/ValidClassNameSniff.php [hunk 1]
```diff
@@ -152,8 +148,6 @@ public function process(File $phpcsFile, $stackPtr)
                 }
                 break;
             }
-
-            $stackPtr++;
```
Comment (mmoll) [inline Symfony/Sniffs/NamingConventions/ValidClassNameSniff.php]: dead code

File: Symfony/Sniffs/NamingConventions/ValidClassNameSniff.php [hunk 2]
```diff
@@ -136,10 +136,6 @@ public function process(File $phpcsFile, $stackPtr)
                 $name = $phpcsFile->findNext(T_STRING, $stackPtr);
                 $function = $phpcsFile->findNext(T_FUNCTION, $stackPtr);
 
-                if (false === $name) {
-                    break;
-                }
```
Comment (mmoll) [inline Symfony/Sniffs/NamingConventions/ValidClassNameSniff.php]: leftover of now uneeded static analysis fix

**Comment by mmoll** [comment]
This leaves only two lines in `AlphanumericFilenameSniff.php` uncovered which would require to make this test more special, so I'd claim "almost" 100% coverage. :)

File: Symfony/Sniffs/Commenting/AnnotationsSniff.php [hunk 3]
```diff
@@ -28,13 +28,7 @@
...
 class AnnotationsSniff implements Sniff
 {
-
-    /**
-     * Pattern
-     *
-     * @var string
-     */
-    private static $_pattern = '/^@([^\\\(]+).*$/i';
+    const PATTERN = '/^@([^\\\(]+).*$/i';
```
Comment (mmoll) [inline Symfony/Sniffs/Commenting/AnnotationsSniff.php]: Probably a PHPUnit/xdebug problem that this is not claimed covered in the `preg_replace` calls, but it's IMHO cleaner as a constant anyway...

**Comment by djoos** [comment]
Super - thanks for your contribution @mmoll!

**Pull request merged**